### DISCUSSION TITLE
添加鼠标侧键前进后退功能

### DISCUSSION THREAD

**Discussion by KirinStar-t** [Coolapk-UWP/Coolapk-UWP#171] [Q&A]
更新v3版了，但是好像没有鼠标侧键前进后退功能，我记得v2是有的，现在总是按侧键返回不生效，有点不习惯😅

**Comment by wherewhere** [comment]
从来也没有这个功能，侧键是自定义键，我不知道你的侧键是什么

**Reply by KirinStar-t** [reply]
有可能v2版是相当于网页的形式吧，我的侧键就是前进后退的功能，比如在文件管理器、浏览器上面都可以起作用

**Reply by wherewhere** [reply]
可能是win11把快捷方式改掉了，win10上快捷方式是win+back，UWP只会响应win+back，不会响应新的快捷方式

**Reply by KirinStar-t** [reply]
可是我之前也是用win11的😂
我搜了一下，这两个侧键分别叫前侧键（前进键）、后侧键（返回/后退键）代码好像是XButton1、XButton2
![image](https://user-images.githubusercontent.com/112538497/232839159-f24e15b5-7f87-4688-b185-746f72ab9866.png)

**Reply by wherewhere** [reply]
反正我不会增加快捷方式，防止在其他平台上出问题，现在已经发现17763炸了
